### PR TITLE
Chore/initial version

### PR DESCRIPTION
**Pull request by santiagonoguez**
Most of this code was already existent, not new:

- Initial CCI config version, based on others; there's a card later for CCI so I didn't invest much time here.
- Based configuration on previous effort
- The component is a mix of rise-image and rise-data-image. A few functions were renamed but the code of both components is mixed, and I added no extra features.
- The tests are taken from rise-data-image, with just a check added to see if the internal component receives the url. No extra tests were added.
- I just copied the existing demo for testing, no modifications. There's a demo card later to clean or enhance this later.
- Readme files for repository ( based on financial component readme layout ) and demo ( based on previous demo readme file ).

Validation: https://apps.risevision.com/schedules/details/710a1da0-8e90-43f4-9836-f8dfbdb03fae?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

capture

### CODE REVIEW

**Comment by santiagonoguez** [comment]
@stulees @fjvallarino reminder to review this, please.

**Comment by stulees** [comment]
@santiagonoguez Apologies, I'll review by EOD

**Comment by santiagonoguez** [comment]
@stulees I'll simplify it

**Comment by santiagonoguez** [comment]
@stulees I removed all code related to styling and the iron related properties of rise-image;  I retained code related to rise-data-image and tests. I removed README entries related to styling and iron related properties. Please tell me if there is something additional that you consider it shouldn't be here. Thanks !

File: src/rise-image.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+  static get STORAGE_PREFIX() {
+    return "https://storage.googleapis.com/";
+  }
+
+  // Each item of observers array is a method name followed by
+  // a comma-separated list of one or more dependencies.
+  static get observers() {
+    return [
+      "_handlePlaceholder(placeholder)"
```
Comment (stulees) [inline src/rise-image.js]: @santiagonoguez This can be removed

File: src/rise-image.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+  _getComponentData() {
+    return {
+      name: "rise-image",
+      id: this.id,
+      version: version
+    };
+  }
+
+  _handlePlaceholder( value ) {
```
Comment (stulees) [inline src/rise-image.js]: @santiagonoguez This can be removed

File: src/rise-image.js (hunk 1 above)
Comment (santiagonoguez) [reply src/rise-image.js]: @stulees removed

File: src/rise-image.js (hunk 2 above)
Comment (santiagonoguez) [reply src/rise-image.js]: @stulees removed

**Pull request merged**